CHANGELOG.md

```diff
@@ -12,6 +11,9 @@
 ## 1.21.16 (Not yet released)
 * Support for .NET Core 3.1 in csx files (PR: [#1731](https://github.com/OmniSharp/omnisharp-roslyn/pull/1731))
 * Update the minimal MSBuild to better support .NET 5 Previews ([omnisharp-vscode#3653](https://github.com/OmniSharp/omnisharp-vscode/issues/3653), PR: [#1746](https://github.com/OmniSharp/omnisharp-roslyn/pull/1746))
+* Razor support for `<text>` tag completions.
+* Ability to restart the Razor Language Server to activate changes to the `razor.trace` level.
+* Bug fixes and performance improvements.
```

Comment:
We have a pre-release build of 1.21.16 for .NET 5 Preview 2 validation which will likely be released on the 30th. 

```suggestion
## 1.21.17 (Not yet released)
* Updated Razor support (PR:[#3696](https://github.com/OmniSharp/omnisharp-vscode/pull/3696))
  * Razor support for `<text>` tag completions.
  * Ability to restart the Razor Language Server to activate changes to the `razor.trace` level.
  * Bug fixes and performance improvements.

## 1.21.16 (March 30, 2020)
* Support for .NET Core 3.1 in csx files (PR: [#1731](https://github.com/OmniSharp/omnisharp-roslyn/pull/1731))
* Update the minimal MSBuild to better support .NET 5 Previews ([omnisharp-vscode#3653](https://github.com/OmniSharp/omnisharp-vscode/issues/3653), PR: [#1746](https://github.com/OmniSharp/omnisharp-roslyn/pull/1746))
```